package.json

```diff
@@ -34,6 +34,7 @@
   "dependencies": {
     "async": "^0.9.0",
     "bcryptjs": "^2.1.0",
+    "bluebird": "^2.9.9",
```

Comment:
Please use `3.x` version - the same as we do in juggler (https://github.com/strongloop/loopback-datasource-juggler/pull/790)